xapian-bindings/java/SmokeTest.java

```diff
@@ -142,6 +153,14 @@ public static void main(String[] args) throws Exception {
 		System.err.println("Unexpected docid");
 		    System.exit(1);
 	    }
+	    // New iterator, should start from the beginning 
+	    int iterations = 0;
+	    for(MSetIterator.MatchDescriptor descriptor : mset) {
```

Comment:
I wanted to create a test where there were more documents, but currently there is only 1 document. I didn't think it was a good idea to add more documents now because the tests are somewhat dependent on each other. After switching to jUnit we won't have to worry about the dependencies.